website/content/community-plugins.mdx

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Should these pages be part of the site navigation? I don't see them on packer.io at the moment but they are reachable.

Reply:
I'm not sure - I retained the site's existing structure for this PR but the nav can be changed any time via `website/data/subnav.js` 😀